app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WelcomeController < ApplicationController
```

Comment:
This is to test functionality of heroku, will change when building out welcome page

Reply:
this is great since Jack and I need it to in order to test the nav bar and display on local host!

Reply:
Great thinking Eric!